.gitignore

```diff
@@ -34,3 +34,6 @@ test_certs
 
 # ld scripts
 layout.ld
+
+# cargo nextest test archive
+caliptra-test-bins.tar.zst
```

Comment:
```suggestion
**/*.tar.zst
```
May as well avoid checking in any Zstandard archives.

Reply:
fixed.